### ISSUE TITLE
Add in memo support for transfers

### ISSUE THREAD

**Issue by Vheissu** [NeblioTeam/neblio#329]
When transferring NEBL or any other tokens, there is no ability to add in a memo. This would be a great feature to have so people can see notes with transfers.

**Comment by NeblioTeam** [comment]
NTP1 metadata could carry a memo or any other data, there would just need to be a standardized format for memos so they could be created by the sender and parsed by the recipient's wallets. 

Are you looking for something similar to our NTP1 metadata, but with a regular NEBL transfer, or am I misunderstanding this issue?

**Comment by Vheissu** [comment]
@NeblioTeam This is correct. Basically, the ability for non-NTP1 token transfers to support a memo field, or better still, to support metadata would be great. I have no idea of the work involved, but I assume there is a reason we have non-NTP1 tokens as well as separate NTP1 tokens as the original tokens might not easily be able to support meta, is that right?
